docs/troubleshooting.md

```diff
@@ -2,11 +2,14 @@
 
```

Comment:
This can go too - this is now a troubleshooting section on the webpage, linking to that document above in the `README.md` is sufficient 👍